### ISSUE TITLE
Search term listing all with that word

### ISSUE THREAD

**Issue by meowbands** [Paturages/chorus#273]
When you search for something, it's not a one-and-done search only THIS. It includes anything that has that term in it. It's the most noticeable in the genre search tag, since in other tags they can be specified. If you do 'genre="Pop"' it will bring up all genres containing pop, like Pop Rock, Europop, and J-Pop. There's no way to exclude other terms, it seems like, either, like a special character to only bring up Pop. 

Not sure if this is a bug or intentional, but I figure I'd bring it to your attention. Sorry if it's not clear!

**Comment by Paturages** [comment]
Yeah that's intentional, to be able to search song names/artists by partial name. I most likely should add some kind of option for exact matches.
